public/src/js/widgets/columns/fronts-config.html

```diff
@@ -126,7 +126,10 @@
              <!-- ko if: id -->
                 <div class="instructions">
                     <!-- ko if: !state.isOpenProps() && collections.items().length -->
-                        <span class="linky tool--metadata" data-bind="click: openProps">edit metadata</span>
+                        <span class="linky tool--metadata" data-bind="
+                          click: openProps,
+                          visible: !state.isMetaLocked
```

Comment:
Yes it works, but I guess because it is not an observable. I tried to replicate the behaviour of the isTypeLocked property, so shall I change that to an observable as well?